spec/rspec/core/runner_spec.rb

```diff
@@ -159,24 +159,30 @@ def interrupt
     # interface.
     describe ".running_in_drb?", :slow do
       it "returns true if drb server is started with 127.0.0.1" do
-        allow(::DRb).to receive(:current_server).and_return(double(:uri => "druby://127.0.0.1:0000/"))
+        allow(::DRb).to receive(:current_server).and_return(double(:uri => "druby://127.0.0.1:0000/", :alive? => true))
```

Comment:
While you're modifying these specs, is it worth changing from `double` to `instance_double`? (e.g. `instance_double(::DRb::Server)` or whatever the class is?)